spec/classes/collectd_plugin_modbus_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+
+describe 'collectd::plugin::modbus', type: :class do
```

Comment:
```suggestion
describe 'collectd::plugin::modbus' do
```
The note about the type isn't required anymore (but it also doesn't hurt)

Reply:
np, let me get rid of it